doc/guide/build.hpp

```diff
@@ -209,17 +209,27 @@ The full list of options mlpack allows:
 Each option can be specified to CMake with the '-D' flag.  Other tools can also
 be used to configure CMake, but those are not documented here.
 
+For example, if you would like to build mlpack and its CLI binding statically, then
```

Comment:
```suggestion
For example, if you would like to build mlpack and its CLI bindings statically, then
```
Minor pedantry. :)

Reply:
Also, do you think you can update the `BUILD_SHARED_LIBS` documentation above?  Right now it just mentions libraries, but it should be "libraries and executables".

Reply:
Yeah, of course